src/utils/validadeMediaAndText/index.spec.ts

```diff
@@ -0,0 +1,133 @@
+import {
+  validateImageAspectRatio,
+  validateImageResolution,
+  validateMediaSize,
+  validateTextLength,
+  validateVideoDuration,
+  validateVideoResolution,
+} from '.';
+
+const mockFileImage = new File([new ArrayBuffer(10 * 10 + 1)], 'mockfile.jpg', {
+  type: 'image/jpeg',
+});
+
+const mockFileVideo = new File([new ArrayBuffer(10 * 10 + 1)], 'mockfile.mp4', {
+  type: 'video/mp4',
+});
+
+describe('validate texts', () => {
+  it('validate text length', () => {
+    const limiteLength = 2000;
+
+    expect(validateTextLength('Bom dia!', limiteLength)).toBe(true);
+  });
+});
+
+describe('validate medias', () => {
+  it('validate media size', () => {
+    const limitSize = 9999;
+
+    expect(validateMediaSize(mockFileImage, limitSize)).toBe(true);
+  });
+});
```

Comment:
O seu describe pode ser um grande `describe` que dentro tem dentro as validações expecificas tipo

```js
describe('Validate', () => {
  describe('input text', () => {
     it('max lenght', () => {})
  })
})
```

lembre-se que seus `describe`s e `it`s, devem conversar na hora do output do teste para ficar legivel e intendivel